### PR TITLE
Test/64/test sliderinput

### LINKED ISSUE #64: (SP: 1) Write tests for slider with input component
Writing tests don't forget to mock elements if neccessary.

- [ ] it should renders correctly
- [ ] it should call onChange when slider is moved
- [ ] it should update inputValue correctly when input value is empty
- [ ] it should update prices when input is blurred and input is greater than max value
- [ ] it should not update prices when input is blurred and value in input has not changed

### PR DESCRIPTION
**Pull request by mistercapxx**
#
Screenshot 2024-07-18 at 20 14 02
64 - Slider With Input tests ready

### CODE REVIEW

File: src/constants/translations/en/guest-home-page.json
Comment (markgol777) [inline src/constants/translations/en/guest-home-page.json:1]: I'm pretty sure for the issue linked you didn't need to change this file.

File: src/tests/unit/components/slider-with-input/SliderWithInput.spec.jsx
Comment (markgol777) [inline src/tests/unit/components/slider-with-input/SliderWithInput.spec.jsx:1]: everything else looks good here

**Review by Yaroslav-Yakovlev — COMMENTED** [review]
You can use beforeEach method in describe block to configure the state of execution of each test.

**Review by Yaroslav-Yakovlev — APPROVED** [review]

**Review by kryzanivska-nastya — APPROVED** [review]

**Review by miller-113 — APPROVED** [review]

**Review by CallMeMaverick — APPROVED** [review]

File: src/tests/unit/components/slider-with-input/SliderWithInput.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    await waitFor(() => {
+      expect(mockOnChange).toHaveBeenCalled()
+    })
+  })
+
+  test('update inputValue correctly when input value empty', async () => {
+    const inputElement = screen.getByRole('textbox')
+    userEvent.clear(inputElement)
+    userEvent.tab()
```
Comment (nrevutska) [inline src/tests/unit/components/slider-with-input/SliderWithInput.spec.jsx]: await is missed here (should be the same as in 38 row)

**Review by nrevutska — CHANGES_REQUESTED** [review]
add awaits everywhere where it is needed

**Review by nrevutska — CHANGES_REQUESTED** [review]
fix formatting please
image

**Review by markgol777 — APPROVED** [review]
good

**Review by nrevutska — APPROVED** [review]

**Pull request merged**